### PR TITLE
Cmrarc 789 - logic to call and process pyquarc api

### PR DESCRIPTION
**Pull request by cgokey**

### CODE REVIEW

File: lib/file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"hello"}
```
Comment (tle1989) [inline lib/file.txt:1]: This file was created only for testing purpose right?

File: lib/dashboard_checker.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import sys
 import json
 
-# Uses pyQuARC 1.1.5
+# Uses pyQuARC 2.0
```
Comment (tle1989) [inline lib/dashboard_checker.py:8]: Maybe 2.0.0 just to have the standard format for future reference.

**Review by tle1989 — APPROVED** [review]

File: lib/file.txt (hunk 1 above)
Comment (cgokey) [reply lib/file.txt:1]: Good catch I'll remove this.

File: lib/dashboard_checker.py (hunk 2 above)
Comment (cgokey) [reply lib/dashboard_checker.py:8]: Yeah, it is not that important for code, but it is for bamboo, but hopefully next PI we'll do away with the command line altogether.

Comment (cgokey) [reply lib/dashboard_checker.py:8]: Actually I'm going to try and remove these files as I don't think they are needed anymore

**Review by dmistry1 — APPROVED** [review]

**Review by htranho — APPROVED** [review]

**Review by rosycordova — APPROVED** [review]

**Pull request merged**